### PR TITLE
Remove locations.yml and related

### PR DESCRIPTION
**Pull request by hnformentin**
Resolves https://github.com/equinor/komodo/issues/364

I am not sure about how to test if something is being broken by these PRs other than the tests of the repos.
Any ideas?

### CODE REVIEW

File: komodo/cli.py [hunk 1]
```diff
@@ -113,17 +113,6 @@ def _main(args):
 
     create_enable_scripts(komodo_prefix=tmp_prefix, komodo_release=args.release)
 
-    if args.locations_config is not None:
-        with open(args.locations_config, mode="r", encoding="utf-8") as defs, open(
-            Path(args.release) / "local", mode="w", encoding="utf-8"
-        ) as local_activator, open(
```
Comment (xjules) [inline komodo/cli.py:119]: Just out of curiosity what's the difference between `locations` and `local_activators`?

**Comment by xjules** [comment]
> Resolves #364
> 
> I am not sure about how to test if something is being broken by these PRs other than the tests of the repos. Any ideas?

I would just double check [komodoenv](https://github.com/equinor/komodoenv) if there is no usage of locations, if that is so, it can go in :)

**Comment by berland** [comment]
There are two files `local.*`  in `komodo/data` which can also be purged.

File: komodo/cli.py (hunk 1 above)
Comment (hnformentin) [reply komodo/cli.py:119]: I am not sure. I removed it because `args.locations_config is not None:` will never be true without locations.

**Comment by hnformentin** [comment]
> > Resolves #364
> > I am not sure about how to test if something is being broken by these PRs other than the tests of the repos. Any ideas?
> 
> I would just double check [komodoenv](https://github.com/equinor/komodoenv) if there is no usage of locations, if that is so, it can go in :)

I did not find any instance of `locations` in komodoenv repo, but now I found of `local`, which I think should also be removed: https://github.com/equinor/komodoenv/pull/55

File: komodo/cli.py (hunk 1 above)
Comment (berland) [reply komodo/cli.py:119]: the "local activator"s are the shell scripts that are run by the enable script, so they should be removed yes.

File: tests/test_cli.py [hunk 2]
```diff
@@ -143,12 +141,7 @@ def test_pyver_is_deprecated(capsys):
     """
     pkgs = os.path.join(_get_test_root(), "data/cli/nominal_release.yml")
     repo = os.path.join(_get_test_root(), "data/cli/nominal_repository.yml")
-    locs = os.path.join(_get_test_root(), "data/cli/locations.yml")
-    cmd = (
-        f"{pkgs} {repo} --prefix pfx --release rel "
-        f"--locations-config {locs} "
-        f"--pyver 3.8"
-    )
+    cmd = f"{pkgs} {repo} --prefix pfx --release rel " f"--pyver 3.8"
```
Comment (kwinkunks) [inline tests/test_cli.py:144]: Don't really need string concatenation here.

**Comment by kwinkunks** [comment]
FWIW, I think all this looks good -- much need work, nice one!

I have not read in detail, but if no-one else does it, I can look a little deeper and approve.

File: tests/test_cli.py (hunk 2 above)
Comment (hnformentin) [reply tests/test_cli.py:144]: I don't understand your comment, could you please make a suggestion on change required on your view?

**Comment by hnformentin** [comment]
> FWIW, I think all this looks good -- much need work, nice one!
> 
> I have not read in detail, but if no-one else does it, I can look a little deeper and approve.

Please, need help to review if someone (e.g. you) volunteers :-)

File: tests/test_cli.py (hunk 2 above)
Comment (xjules) [reply tests/test_cli.py:144]: I think @kwinkunks refers to joining those two strings: 
`cmd = f"{pkgs} {repo} --prefix pfx --release rel --pyver 3.8"`

**Review by xjules — CHANGES_REQUESTED** [review]
Just had a small comment (which is to forwarding Matt's comment :smile: ), but looks very good! Nice job @hnformentin

File: tests/test_cli.py (hunk 2 above)
Comment (hnformentin) [reply tests/test_cli.py]: I see, thanks for pointing out. Fixed.

**Comment by hnformentin** [comment]
Thanks, I fixed the requested change now :-)

**Review by xjules — APPROVED** [review]
Good job @hnformentin ! :rocket:

**Pull request merged**